Dockerfile.no-systemd

```diff
@@ -0,0 +1,35 @@
+FROM balenalib/amd64-debian:buster-run
+
+ENV TERM xterm
+
+# Install ops/sre related packages.
+RUN install_packages \
+		htop \
+		jq \
+		nano \
+		netcat \
+		net-tools \
+		ifupdown \
+		strace \
+		vim \
+		wget
+# And configure them.
+COPY src/htoprc /root/.config/htop/
+
+# We use NodeJs everywhere.
+ENV NODE_VERSION 10.16.3
+ENV NPM_VERSION 6.10.3
+RUN curl -SL "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" | tar xz -C /usr/local --strip-components=1 \
+	&& npm install -g npm@"$NPM_VERSION" \
+	&& npm cache clear --force \
+	&& rm -rf /tmp/*
+
+# Confd binary will be removed once we migrate to a different configuration management.
+ENV CONFD_VERSION 0.16.0
+RUN wget -O /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-amd64 \
+	&& chmod a+x /usr/local/bin/confd
+ONBUILD COPY config/confd /etc/confd
```

Comment:
This respects the current conventions of keeping confd templates in `<project>/config/confd`.  So templates will be copied to the final image without extra instructions and will be used by the entry point script.